tests/tasks/assert_profile_present.yml

```diff
@@ -1,12 +1,17 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
 - include_tasks: get_profile_stat.yml
-- name: "assert that the profile is present - '{{ profile }}'"
+- name: Assert that profile is present - {{ profile }}
   assert:
     that: lsr_net_profile_exists
     msg: "profile {{ profile }} does not exist"
 
-- name: "assert that the ansible managed comment is present in '{{ profile }}'"
+- name: Assert that the ansible managed comment is present in {{ profile }}
   assert:
     that: lsr_net_profile_ansible_managed
     msg: "profile {{ profile }} does not have the ansible managed comment"
```

Comment:
I think @liangwen12year is already working on this. Could you drop this changes? I think it is better to have all the ansible-lint fixes in separated commits so it is easier to track. Other than that, LGTM

Reply:
Done!